themes/barbuk/barbuk.theme.bash

```diff
@@ -1,110 +1,112 @@
 # shellcheck shell=bash
 # shellcheck disable=SC2034 # Expected behavior for themes.
-# shellcheck disable=SC2154 #TODO: fix these all.
 
 # Theme custom glyphs
-SCM_GIT_CHAR_GITLAB=${BARBUK_GITLAB_CHAR:='  '}
-SCM_GIT_CHAR_BITBUCKET=${BARBUK_BITBUCKET_CHAR:='  '}
-SCM_GIT_CHAR_GITHUB=${BARBUK_GITHUB_CHAR:='  '}
-SCM_GIT_CHAR_DEFAULT=${BARBUK_GIT_DEFAULT_CHAR:='  '}
-SCM_GIT_CHAR_ICON_BRANCH=${BARBUK_GIT_BRANCH_ICON:=''}
-SCM_HG_CHAR=${BARBUK_HG_CHAR:='☿ '}
-SCM_SVN_CHAR=${BARBUK_SVN_CHAR:='⑆ '}
-EXIT_CODE_ICON=${BARBUK_EXIT_CODE_ICON:=' '}
-PYTHON_VENV_CHAR=${BARBUK_PYTHON_VENV_CHAR:=' '}
-COMMAND_DURATION_ICON=${BARBUK_COMMAND_DURATION_ICON:-"$bold_blue  "}
+: "${BARBUK_GITLAB_CHAR:='  '}"
+: "${BARBUK_BITBUCKET_CHAR:='  '}"
+: "${BARBUK_GITHUB_CHAR:='  '}"
+: "${BARBUK_GIT_DEFAULT_CHAR:='  '}"
```

Comment:
```
< SCM_GIT_CHAR_DEFAULT=${BARBUK_GIT_DEFAULT_CHAR:='  '}
> : "${BARBUK_GIT_DEFAULT_CHAR:='  '}"
```

With these changes, `BARBUK_GIT_DEFAULT_CHAR` still gets its defaulting assignment, but where in the code does `SCM_GIT_CHAR_DEFAULT` come to have the value of `BARBUK_GIT_DEFAULT_CHAR` ?